src/utils/theme.js

```diff
@@ -36,7 +36,19 @@ const defaultThemeObject = {
   },
   typography: {
     fontSize: 14,
-    fontFamily: '"Helvetica Neue", Helvetica, Arial, sans-serif',
+    fontFamily: 'Roboto, "Helvetica Neue", Helvetica, Arial, sans-serif',
```

Comment:
We'll need to also add font source for Roboto as we have for Poppins:
https://mui.com/material-ui/react-typography/#install-with-npm

Reply:
Good catch, thanks!